draft-ietf-quic-transport.md

```diff
@@ -621,10 +621,18 @@ packet. It also sets the Token field to the token provided in the Retry. The
 client MUST NOT change the Source Connection ID because the server could include
 the connection ID as part of its token validation logic (see {{tokens}}).
 
+The Initial packet sent by the client in response to a Retry packet is subject
+to the same restrictions as the first Initial packet, with the exception of the
+value of the Destination Connection ID and Token fields, which are set as
+described here.  A client can either reuse the cryptographic handshake message
```

Comment:
How about putting the last sentence in this paragraph first, then saying "Aside from this, the Initial packet sent by the client is subject to the same restrictions as the first Initial packet."